.github/workflows/.build.yml

```diff
@@ -259,9 +259,6 @@ jobs:
           for tag in $(jq -r '.tags[]' <<< "$DOCKER_METADATA_OUTPUT_JSON"); do
             echo "* \`${tag}\`" >> /tmp/tags.txt
           done
```

Comment:
Nit: perhaps move this block as well to keep them a bit more together?

Reply:
Actually wondering now if the intermediate `tags.txt` is needed at all if it's gonna be appended to `/tmp/summary.txt` or are both used?

Reply:
Moved